pom.xml

```diff
@@ -175,6 +170,14 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-invoker-plugin</artifactId>
           <version>2.0.0</version>
+          <configuration>
+            <extraArtifacts>
+              <extraArtifact>org.jacoco:org.jacoco.agent:0.7.9:jar:runtime</extraArtifact>
+            </extraArtifacts>
+            <environmentVariables>
+              <MAVEN_SKIP_RC>1</MAVEN_SKIP_RC>
+            </environmentVariables>
+          </configuration>
```

Comment:
some explanations ...

for jacoco to analyze integration test runs, it has to be in the local repository used by the invoker plugin when runngin tests, so that the agent is found and run. The `[extraArtifacts](http://maven.apache.org/plugins/maven-invoker-plugin/install-mojo.html#extraArtifacts)` parameter allows to copy that jar to the local repository

In the `pre-integration-test` we call the `prepare-agent-integration` goal of Jacoco (see below) to create the `invoker.mavenOpts` variable, containing what is needed for the agent to run during the integration tests. This variable is automatically used [by the invoker plugin](http://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#mavenOpts) to launch its internal maven commands. But this only works it there is no `.mavenrc` file or if this file is ignored. If there is this file, it is taken into account, and the configuration on the variable is ignored. Thus this environment variable `MAVEN_SKIP_RC` to skip any `.mavenrc` file to override what the jacoco agent defined in the `invoker.mavenOpts` variable.